host_vars/dalite.harvardx.harvard.edu/public.yml

```diff
@@ -0,0 +1,47 @@
+---
```

Comment:
@antoviaque This has been addressed in 773dbd5156111ac3ee98503128449bc64202521e. No variables that we don't want to share between all instances are left in this file.